app/models/concerns/labelable.rb

```diff
@@ -13,4 +13,9 @@ def add_labels(new_labels = nil)
     new_labels << labels
     update!(label_list: new_labels)
   end
+
+  def add_automation_labels(new_labels = nil)
```

Comment:
we already have the `add_label` method right?

Reply:
I just wanted to add `labels.any?` check to add_labels, thought it shouldn't affect the existing functionality so added a new method.